File: Scala/Example7.scala
object Example7 extends App {
    //Function as parameter
    def two(x : Int) = x * 2
    def three(x : Int) = x * 3

    def fun(x : Int, f : (Int) => Int) = f(x)

    println(fun(4, two)) // >> 8
    println(fun(4, three)) // >> 12
}

File: С#/Example7.cs
using System;

class Example7 {
	static void Main() {
		int[] arr_1 = {1, 2, 3, 4, 5};
		int[,] arr_2 = {{1, 2}, {3, 4}};

		// Now we can iterate through elements in one-dimensional array
		foreach (int elem in arr_1)
			Console.Write("{0}, ", elem);
		Console.WriteLine("\n");

		// We can iterate through elements in two-dimensional array
		foreach (int elem in arr_2)
			Console.Write("{0}, ", elem);
		Console.WriteLine("\n");
	}
}